src/components/LoginBox.tsx

```diff
@@ -0,0 +1,99 @@
+import styled from "styled-components";
+import { useState } from "react";
+import { Button, TextField } from "@material-ui/core";
+import { logIn } from "../util/loginCookies";
+
+const LoginRoot = styled.div`
+  margin-bottom: 2rem;
+  margin-top: 4rem;
+  justify-content: center;
+  flex-direction: column;
+  justify-items: center;
+`;
+
+const LoginContainer = styled.div`
+  height: 500px;
+  width: 500px;
+  text-align: center;
+  padding: 15px;
+  background-color: grey;
+`;
+
+const LoginFields = styled.div`
+    margin-top: 65px;
+    padding: 15px;
+`;
+
+const LoginButton = styled(Button)`
+  && {
+    background-color: #5dc0cb;
+    display: block-inline;
+    margin-top: 45px;
+  }
+`;
+
+function LoginBox(): JSX.Element {
+    const [name, setName] = useState('');
+    const [pass, setPass] = useState('');
+    const [nameErr, setNameErr] = useState('');
+    const [passErr, setPassErr] = useState('');
+
+    function ValidateLogin(){
```

Comment:
As of now, yes. There is still a 3-63 characters range for usernames and password because of how Azure stores them. Validation for this wouldn't be necessary on the login page as it will be needed later when creating a registration page.